app/controllers/application_controller.rb

```diff
@@ -13,10 +13,4 @@ class ApplicationController < ActionController::Base
   def after_sign_in_path_for(resource)
     stored_location_for(resource) || resource.after_sign_in_path || super
   end
-
-  protected
-
-  def find_selectable_weeks
-    @current_weeks = Week.selectable_from_now_until_end_of_period
```

Comment:
not really understandable ; too much magic here. it is set directly in the controller.